editor/editor_properties.h

```diff
@@ -755,7 +755,6 @@ class EditorPropertyResource : public EditorProperty {
 	EditorPropertyResource();
 };
 
-///////////////////////////////////////////////////
 /// \brief The EditorInspectorDefaultPlugin class
 ///
```

Comment:
```suggestion
// The EditorInspectorDefaultPlugin class
```

Since this is the only place `\brief` is used, probably better to remove it as well to avoid confusion.